tools/operator/testdata/kafka-couchdb-tls/sbecc-input.yml

```diff
@@ -0,0 +1,99 @@
+tls: true
+organizations:
+  - name: org1
+    connProfilePath: ./connection-profile
+  - name: org2
+    connProfilePath: ./connection-profile
+  - name: org3
+    connProfilePath: ./connection-profile
+  - name: org4
+    connProfilePath: ./connection-profile
+
+installChaincode:
+  - name: sbecc
+    version: v1
+    path: github.com/hyperledger/fabric-test/chaincodes/sbe/go
+    organizations: org1,org2,org3,org4
+    language: golang
+    metadataPath: ""
+
+instantiateChaincode:
+  - channelName: testorgschannel3
+    name: sbecc
+    version: v1
+    args: ""
+    organizations: org1
+    endorsementPolicy: ""
+    collectionPath: ""
+
+  - channelName: testorgschannel4
+    name: sbecc
+    version: v1
+    args: ""
+    organizations: org3
+    endorsementPolicy: ""
```

Comment:
For  both chan3 and chan4, the SBE transactions are sent to two orgs, but your endorsement policies should also be set here to be 2 of (peer in org1, peer in org2), and 2 of (peer in org3, peer in org4). For example, refer to the "deploy" section in github.com/hyperledger/fabric-test/tools/PTE/CITest/FAB-11615-2i/preconfig/sbe_cc/sbecc-chan1-instantiate-TLS.json

Reply:
I will change this in systemtest to ginkgo PR.